src/tags_int.hpp

```diff
@@ -174,6 +174,7 @@ namespace Exiv2 {
         sony2CsId,
         sony2Cs2Id,
         sony2FpId,
+		sonyMisc1Id,
```

Comment:
Please could you fix this indentation?

Reply:
I was caught out by Eclipse's formatting the code. I have made the change and altered Eclipse's preferences.